ament_clang_format/package.xml

```diff
@@ -9,6 +9,11 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>python3-setuptools</build_depend>
+
+  <buildtool_depend>python3</buildtool_depend>
```

Comment:
I don't see why `python3-dev` would be needed "in most cases".

But I don't think we should require every Python package to declare a dependency on `python3`. As mentioned before we also don't declare a dependency on `build-essential` when  building a package which needs a compiler.